src/index.html

```diff
@@ -7,8 +7,62 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header 
+      class="header"
+    >
+      <img 
+        src="./images/logo.png" 
+        alt="logo"
+        class="logo"
+      >
+      <nav
+        class="nav"
+      >
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#laptops&computers" class="nav__link"
+              data-qa="hover"
+            >
+              Laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+          
+        </ul>
```

Comment:
"Add empty lines between multiline sibling blocks of HTML" - yes, you did right (but not for first and last element)

```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">Apple</a>
          </li>

          <li class="nav__item">
            <a href="#samsung" class="nav__link">Samsung</a>
          </li>

          <li class="nav__item">
            <a href="#smartphones" class="nav__link">Smartphones</a>
          </li>

          <li class="nav__item">
            <a href="#laptops&computers" class="nav__link"
              data-qa="hover"
            >
              Laptops & computers
            </a>
          </li>

          <li class="nav__item">
            <a href="#gadgets" class="nav__link">Gadgets</a>
          </li>

          <li class="nav__item">
            <a href="#tablets" class="nav__link">Tablets</a>
          </li>

          <li class="nav__item">
            <a href="#photo" class="nav__link">Photo</a>
          </li>

          <li class="nav__item">
            <a href="#video" class="nav__link">Video</a>
          </li>
        </ul>
```